<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>791</width>
      <height>571</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>641</width>
       <height>461</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>0</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:34px;</string>
     </property>
     <property name="text">
      <string>Dr App</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConsulta">
    <property name="title">
     <string>Consulta</string>
    </property>
    <addaction name="action_crearconsulta"/>
    <addaction name="actionBuscar_4"/>
    <addaction name="actionModificar"/>
    <addaction name="actionEliminar"/>
   </widget>
   <widget class="QMenu" name="menuClientes">
    <property name="title">
     <string>Clientes</string>
    </property>
    <addaction name="action_cliente"/>
   </widget>
   <widget class="QMenu" name="menuInventario">
    <property name="title">
     <string>Doctores</string>
    </property>
    <addaction name="action_doctor"/>
   </widget>
   <widget class="QMenu" name="menuReportes">
    <property name="title">
     <string>Reportes</string>
    </property>
    <addaction name="action_reportes"/>
   </widget>
   <widget class="QMenu" name="menuAcerca_de">
    <property name="title">
     <string>Acerca de</string>
    </property>
    <addaction name="action_acerca"/>
   </widget>
   <addaction name="menuConsulta"/>
   <addaction name="menuClientes"/>
   <addaction name="menuInventario"/>
   <addaction name="menuReportes"/>
   <addaction name="menuAcerca_de"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_crearconsulta">
   <property name="text">
    <string>Crear</string>
   </property>
  </action>
  <action name="actionBuscar">
   <property name="text">
    <string>Buscar</string>
   </property>
  </action>
  <action name="action_cliente">
   <property name="text">
    <string>Mantenimiento</string>
   </property>
  </action>
  <action name="actionBuscar_2">
   <property name="text">
    <string>Buscar</string>
   </property>
  </action>
  <action name="action_doctor">
   <property name="text">
    <string>Mantenimiento</string>
   </property>
  </action>
  <action name="actionBuscar_3">
   <property name="text">
    <string>Buscar</string>
   </property>
  </action>
  <action name="action_reportes">
   <property name="text">
    <string>Ver</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="action_facturas">
   <property name="text">
    <string>Facturas</string>
   </property>
  </action>
  <action name="action_inventarios">
   <property name="text">
    <string>Inventario</string>
   </property>
  </action>
  <action name="actionBuscar_4">
   <property name="text">
    <string>Buscar</string>
   </property>
  </action>
  <action name="actionModificar">
   <property name="text">
    <string>Modificar</string>
   </property>
  </action>
  <action name="actionEliminar">
   <property name="text">
    <string>Eliminar</string>
   </property>
  </action>
  <action name="action_acerca">
   <property name="text">
    <string>Ver</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
